test/sysv_sliding_window_test.rb

```diff
@@ -0,0 +1,160 @@
+require 'test_helper'
+
+class TestSysVSlidingWindow < MiniTest::Unit::TestCase
+  CLASS = ::Semian::SysV::SlidingWindow
```

Comment:
This is a little bit of a nitpick, but I'd call this `KLASS` instead of make it a method. `CLASS` is too close to the reserved keyword.